dbt/models/qc/qc.vw_nonlivable_condos_with_chars.sql

```diff
@@ -20,6 +20,3 @@ LEFT JOIN {{ source('spatial', 'township') }} AS towns
 WHERE
     (condos.is_common_area OR condos.is_parking_space)
     AND (condos.char_bedrooms IS NOT NULL OR condos.char_full_baths IS NOT NULL)
-    AND CAST(
-        condos.year AS INT
-    ) BETWEEN {{ var('test_qc_year_start') }} AND {{ var('test_qc_year_end') }}
```

Comment:
This is an unrelated problem I decided to fix while changing references to `test_qc_year_*` variables: Previously this QC report view was relying on hardcoded values for the year filter, which could only change if the view were rerun. To bring it in line with our other QC reports, I'm removing the year filter and requiring us to apply the filter when calling the `scripts/export_models.py` script. Heads up to @wrridgeway who is the de facto owner of this view.